pkg/projectfile/projectfile.go

```diff
@@ -1286,6 +1286,17 @@ func storeProjectMapping(cfg ConfigGetter, namespace, projectPath string) {
 
 			projectPath = filepath.Clean(projectPath)
 
+			for name, paths := range projects {
+				if name == namespace {
+					continue
+				}
```

Comment:
This shouldn't be necessary. Duplicate path entries are duplicate path entries.

Reply:
Without this check, if a `namespace` and `projectPath` are already stored in the config and this function is called, it will first remove that path and then later store it again.

Reply:
I don't see what's the problem with that. Either way you'd end up with the same result, the only difference is that removing this check ensures there's no duplicate path entries.

Reply:
That's fair